chef/site-cookbooks/wca/recipes/default.rb

```diff
@@ -110,7 +110,8 @@
 package 'libmysqlclient-dev'
 package 'imagemagick'
 
-node.default['brightbox-ruby']['version'] = "2.3"
+ruby_version = File.read("#{repo_root}/WcaOnRails/.ruby-version").match(/\d+\.\d+/)[0]
+node.default['brightbox-ruby']['version'] = ruby_version
 include_recipe "brightbox-ruby"
 gem_package "rails" do
   version "4.2.1"
```

Comment:
Holy crap, what is this. This should not be here. I've created https://github.com/thewca/worldcubeassociation.org/issues/1469 to look into this. If you have time to try it out, feel free to take the issue from me.